frontend/src/App.test.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import App from "./App";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import Navbar from "./components/navbar";
 
-test("renders learn react link", () => {
-  render(<App />);
+test("renders all links", () => {
+  render(
+    <Router>
+      <Navbar />
+    </Router>,
+  );
+
+  const homeLink = screen.getByText(/HOME/i);
+  expect(homeLink).toBeInTheDocument();
+
+  const fileUploadLink = screen.getByText(/FILE UPLOAD/i);
+  expect(fileUploadLink).toBeInTheDocument();
```

Comment:
You don't have to change anything here as well but there is a shortcut. 
```suggestion
  getByText(/FILE UPLOAD/i);
```
Calling `getByText()` itself already does the equivalent of `expect(fileUploadLink).toBeInTheDocument();` so you can do this in future tests to make life easier